RecoEgamma/EgammaHLTProducers/src/EgammaHLTElectronDetaDphiProducer.cc

```diff
@@ -133,10 +132,8 @@ std::pair<float,float> EgammaHLTElectronDetaDphiProducer::calDEtaDPhiSCTrk(reco:
       EleRelPointPair scAtVtx(theClus->position(), sclTSOS.globalPosition(), bsPosition);
       deltaeta = fabs(scAtVtx.dEta());
       deltaphi = fabs(scAtVtx.dPhi());
-    } else {
-      recoveryForFailingPropagation = true;
     }
-  } else if (useTrackProjectionToEcal_ or recoveryForFailingPropagation) { 
```

Comment:
I'm actually happy with deleting that file. Its out of date and using it is an error. It'll be deleted as part of the big clean up anyways.

Reply:
will see if I have time to look at which ones should be deleted. Sadly a bit busy right now with other things, such as HEM15/16.

Reply:
So that means we just leave it like that and Sam does his cleaning later? Anyway, I took the opportunity to further clean RecoEgamma/ElectronIdentification a little bit.